src/Components/AddTask/AddTask.tsx

```diff
@@ -0,0 +1,175 @@
+// nadia.mohamed.taha166@gmail.com
+// @Password123!
+import axios from "axios";
+import { useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { AuthContext } from "./../../Context/AuthContext";
+import { ToastContext } from "../../Context/ToastContext";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { ProjectContext } from '../../Context/ProjectContext';
+import { FormValues, projectType } from './../../Types/Types';
+
+
+const AddTask: React.FC = () => {
+  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
+  const { getToastValue }: any = useContext(ToastContext);
```

Comment:
you can change this to void, since the function just displays something in the screen